.gitignore

```diff
@@ -2,3 +2,10 @@ MANIFEST
 *~
 \#*
 dist
+.eggs
+*.egg-info/
+.idea
+build
+.cache/
+*/__pycache__/
+*.pyc
```

Comment:
Just FYI, you can usually put this in your global gitignore, and never have to worry about setting this up for each projects.